Kumi.Game/KumiGameBase.cs

```diff
@@ -58,9 +61,15 @@ private void load()
         // Realm Database, Storage, and Charts
         DependencyContainer.Cache(realm = new RealmAccess(Storage));
         DependencyContainer.CacheAs(Storage);
+        
         var defaultChart = new DummyWorkingChart(Audio, Textures);
         DependencyContainer.Cache(chartManager = new ChartManager(Storage, realm, Audio, Resources, Host, defaultChart));
 
+        Chart = new NonNullableBindable<WorkingChart>(defaultChart);
+        
+        DependencyContainer.CacheAs<IBindable<WorkingChart>>(Chart);
```

Comment:
It doesn't really feel right for there to simply just be a chart randomly sitting here. You're already passing `ChartManager` to the music controller, so there's really no reason you can't just add a function to `ChartManager` to get every usable set and iterate through that whenever the overlay is relevant.

Reply:
This singular chart is cached in the dependencies to be used across other drawables that may use it, e.g. playfield. `ChartManager` refreshes realm entirely when we call upon the `GetAllUsableCharts`, which isn't something we want when we change the tracks constantly, only when we initially load the charts in a song selection screen. `ChartManager` is used to fetch a cached (or create a working chart if it isn't cached already) chart so that we can load the song and play it in the menu.

I feel it's better to have a global chart bindable cached in the dependencies rather than having to fetch the manager, since then we won't have to jump through hoops just to fetch the current beatmap through the manager and instead set it directly through the bindable instead.